app/services/mapquest_service.rb

```diff
@@ -0,0 +1,16 @@
+class MapquestService
+  def self.conn
+    conn = Faraday.new(url: "http://www.mapquestapi.com/directions/v2/")
+  end
+
+  def self.get_json(response)
+    JSON.parse(response.body, symbolize_names: true)
+  end
+
+  def self.get_directions(location)
+    destination = NrelFacade.create_fuel_station(location)
```

Comment:
Instead of having to call the NREL facade here (which would make another API call) you could consider, in your controller, passing in the fuel station from the NREL facade call. This would just limit API calls/improve performance